### PR TITLE
Various validation/compatibility/driver fix

### PR DESCRIPTION
**Pull request by philipostli**
Found some issues on my cloud device that I fixed:

- `homey app validate` failed - removed homey-cli & update app minimum compatibility version (to support widgets)
- fixed uri tests not working - changed endpoint /system to /identity
- Added github action for validation & a Readme with a status badge

Hope you find them useful !

### CODE REVIEW

File: lib/PlexAPI.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ module.exports = class PlexAPI extends EventEmitter {
           this.homey.log('[PlexAPI]', `Testing URL ${connection.uri}...`);
 
           const success = await PlexUtil.fetch({
-            url: `${connection.uri}/system`,
+            url: `${connection.uri}/identity`,
```
Comment (WeeJeWel) [inline lib/PlexAPI.js:120]: What didn't work, and why does this new URL work, exactly?

File: README.md
Comment (WeeJeWel) [inline README.md:1]: No need for a README.txt imho.

File: lib/PlexAPI.js (hunk 1 above)
Comment (philipostli) [reply lib/PlexAPI.js:120]: I get 404 response on /system. I find no references to this endpoint in the documentation or any other api-clients. Is this a feature on your specific device perhaps? I find /identity to be working both on local ip and remotly. I have the Plex server running on a Nvidia Shield TV Pro (2019). And this fixed the app for me.

Note: /identity can be called without a plex-token. But is lightweight and good for ping test.

File: README.md
Comment (philipostli) [reply README.md:1]: Its a preference I guess. I find it useful for quick glance at pipeline statuses. We can remove that file of course.

**Review by WeeJeWel — APPROVED** [review]

**Pull request merged**